nss-day-cohort-52/bangazon-api-luciidlou
# the products that store owners can post now have a max selling price …

…of 17500, compared to the original 10000 price cap

### Issue ticket(s): #8 

### New file(s): N/A

### Modified File(s): `models/product.py`

### Modifications:
- Modified the `maxValueValidator` to take `17500` as max price allowed comapared to the orginal max value set at `10000`

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to `al-maxProductPrice`
3. Either inside the client or within Postman, execute a `POST` request on the `products` resource, and set the selling price of the new product to something greater than 17500. You should get an error saying that the price cannot be above 17500. Reset the price to something below 17500. The request should now be a successful.
